application/actions/meetings_test.go

```diff
@@ -15,3 +17,43 @@ func (as *ActionSuite) verifyMeeting(expected models.Meeting, actual api.Meeting
 	as.True(expected.UpdatedAt.Equal(actual.UpdatedAt), msg+", UpdatedAt is not correct")
 	as.Equal(expected.MoreInfoURL.String, actual.MoreInfoURL, msg+", MoreInfoURL is not correct")
 }
+
+func (as *ActionSuite) Test_MeetingsList() {
+	f := createFixturesForMeetings(as)
+	mtgs := f.Meetings
+	lctns := f.Locations
+	user := f.Users[0]
+
+	req := as.JSON("/events")
+	req.Headers["Authorization"] = fmt.Sprintf("Bearer %s", user.Nickname)
+	req.Headers["content-type"] = "application/json"
+	res := req.Get()
+
+	body := res.Body.String()
+	as.Equal(200, res.Code, "incorrect status code returned, body: %s", body)
+
+	wantContains := []string{
+		fmt.Sprintf(`"nickname":"%s"`, user.Nickname),
+		fmt.Sprintf(`"participants":[{"user":{"id":"%s"`, user.UUID.String()),
+		`"url":"http://minio:9000/wca-test-bucket`,
+	}
+	for i := 2; i < 4; i++ {
+		lctn := lctns[i]
+		wantContains = append(wantContains, fmt.Sprintf(`"id":"%s"`, mtgs[i].UUID.String()))
+		wantContains = append(wantContains, fmt.Sprintf(`"name":"%s"`, mtgs[i].Name))
+		wantContains = append(wantContains, fmt.Sprintf(`"start_date":"%s`, mtgs[i].StartDate.Format(domain.DateFormat)))
+		wantContains = append(wantContains, fmt.Sprintf(`"end_date":"%s`, mtgs[i].EndDate.Format(domain.DateFormat)))
+		wantContains = append(wantContains, fmt.Sprintf(`"location":{"description":"%s"`, lctn.Description))
+		wantContains = append(wantContains, fmt.Sprintf(`"country":"%s"`, lctn.Country))
+		wantContains = append(wantContains, fmt.Sprintf(`"latitude":%v`, int(lctn.Latitude.Float64)))
+		wantContains = append(wantContains, fmt.Sprintf(`"longitude":%v`, int(lctn.Longitude.Float64)))
```

Comment:
If you'd like to, you could do something like the following:

```
wantContains = append(wantContains, []string{
			fmt.Sprintf(`"id":"%s"`, requests[i].UUID),
			fmt.Sprintf(`"nickname":"%s"`, creator.Nickname),
			providerJson,
			fmt.Sprintf(`"title":"%s"`, requests[i].Title),
			fmt.Sprintf(`"country":"%s"`, requests[i].Destination.Country),
			fmt.Sprintf(`"description":"%s"`, requests[i].Origin.Description),
			fmt.Sprintf(`"size":"%s"`, requests[i].Size),
			photoJson,
		}...)
```
which I personally find more readable and nicer-looking

Reply:
:+1: